<!DOCTYPE html>
<html lang="en">
<head>
      <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">

    <title>pytube &middot; Domain Analysis of Mosaic Proteins in Purple Sea Urchin</title>

    <link href="/feeds/all.atom.xml" type="application/atom+xml" rel="alternate" title="pytube Full Atom Feed" />
    <link href="/feeds/scipy-2012.atom.xml" type="application/atom+xml" rel="alternate" title="pytube Categories Atom Feed" />

    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css"
          rel="stylesheet" 
          integrity="sha256-7s5uDGW3AHqw6xtJmNNtr+OBRJUlgkNJEo78P4b0yRw= sha512-nNo+yCHEyn0smMxSswnf/OnX6/KwJuZTlNZBjauKhTK0c+zT+q5JOCx0UFhXQ6rJR9jg6Es8gPuD2uZcYDLqSw=="
          crossorigin="anonymous">

    <!-- IE10 viewport hack for Surface/desktop Windows 8 bug -->
    <link href="/theme/css/ie10-viewport-bug-workaround.css" rel="stylesheet">

    <!-- Custom styles for this template -->
    <link href="/theme/css/base.css" rel="stylesheet">

    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
    <!--[if lt IE 9]>
    <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
    <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
    <![endif]-->



    <meta name="tags" content="Bioinformatics Mini-Symposia" />
</head>

<body>
  <div class="container">
    <header id="banner" class="header clearfix">
      <nav>
        <ul class="nav nav-pills pull-right">
            <li role="presentation">
            <a href="/categories.html">Categories</a>
          </li>
        </ul>
      </nav>
      <h3 class="text-muted">
        <a href="/">pytube <strong></strong></a>
      </h3>
    </header>

<section id="content" class="body">
  <header>
    <h2 class="entry-title">
      <a href="/scipy-2012/scipy-2012-domain-analysis-of-mosaic-proteins-in-purple-sea-urchin.html" rel="bookmark"
         title="Permalink to Domain Analysis of Mosaic Proteins in Purple Sea Urchin">Domain Analysis of Mosaic Proteins in Purple Sea Urchin</a></h2>
 
  </header>
  <footer class="post-info">
    <time class="published" datetime="2012-08-31T16:36:00+00:00">
      Fri 31 August 2012
    </time>
    <time class="modified" datetime="2014-04-08T20:28:00+00:00">
      Tue 08 April 2014
    </time>
    <address class="vcard author">
      By           <a class="url fn" href="/author/adam-hughes.html">Adam Hughes</a>
    </address>
  </footer><!-- /.post-info -->
  <div class="entry-content">
    <a class="reference external image-reference" href="https://www.youtube.com/watch?v=l4x23L-mSfk"><img alt="Domain Analysis of Mosaic Proteins in Purple Sea Urchin" src="http://i1.ytimg.com/vi/l4x23L-mSfk/hqdefault.jpg" style="width: 600px;" /></a>
<div class="section" id="description">
<h2>Description</h2>
<p>Purple sea urchins (Strongylocentrotus purpuratus or Sp) are invertebrates
that share more than 7,000 genes with humans, more than other common model
invertebrate organisms like fruit flies and worms. In addition, the innate
immune system of sea urchins demonstrates unprecedented complexity. These
factors make the sea urchin a very interesting organism for investigations of
immunology. Of particular interest are the set of proteins in SP that contain
C-type lectin (CLECT) domains, a functional region in the protein which
recognizes sugars. Proteins containing CLECTs may be particularly important to
immune system robustness because of sugars that are present on pathogens.</p>
<p>The primary goals of this research project are first to identify all the
CLECT-containing proteins in the Sp genome, and then to predict their function
based on similarity to characterized proteins in other species (protein
homology or similarity). The latter goal is particularly challenging and
requires new and creative analysis methods.</p>
<p>From an informational viewpoint, proteins are represented by a unique sequence
of letters, each letter corresponding to an amino acid. For example G-A-V
indicates the sequence glycine, alanine and valine. Commonality between
proteins is usually measured by sequence alignments; that is, by directly
comparing the sequence of letters between two proteins. Algorithms and tools
for these alignments are among the most standardized and available tools in
bioinformatics.</p>
<p>Sequence similarity between homologous proteins can degrade over long
evolutionary timescales. This is in part because some mutations at the
sequence level can occur without compromising a protein's overall function.
This is akin to the evolution of a language, e.g modern English and middle
English, which initially appear to be separate languages due to spelling
differences. Because domains are regions of a protein which can function semi-
independently, they are less prone to accommodate mutations. By comparing
proteins based on the ordering of their domains, or their &quot;domain
architecture&quot;, it becomes possible to identify homology, or similarities in
domain order, separated by extensive evolution.</p>
<p>Alignment tools based on domain architecture are promising, but are still in
their infancy. Consequently, very few researchers utilize both sequence and
domain alignment methodologies corroboratively. Using Python scripts in tandem
with various web tools and databases, we have identified the top alignment
candidates for the CLECT-containing Sp proteins using both methods. With the
help of the Enthought Tool Suite, we have created a simple visualization tool
that allows users to examine the sequence alignments side-by-side with two
types of domain alignments. The information provided by these three results
together is much more informative with respect to predicting protein function
than any single method alone. Finally, we have developed a systematic set of
heuristic rules to allow users to make objective comparisons among the three
sets of results. The results can later be parsed using Python scripts to make
quantitative and qualitative assessments of the dataset. We believe that these
new comparison and visualization techniques will apply in general to
computational proteomics.</p>
</div>
<div class="section" id="details">
<h2>Details</h2>
<ul class="simple">
<li>Category: SciPy 2012</li>
<li>Language: English</li>
<li>Direct Link: <a class="reference external" href="https://www.youtube.com/watch?v=l4x23L-mSfk">https://www.youtube.com/watch?v=l4x23L-mSfk</a></li>
<li>Tags: Bioinformatics Mini-Symposia</li>
</ul>
</div>

  </div><!-- /.entry-content -->
</section>

    <footer class="footer">
      <p>&copy; 2016 pytube</p>
    </footer>

    <script src="https://code.jquery.com/jquery-2.2.0.min.js"></script>
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js" integrity="sha256-KXn5puMvxCw+dAYznun+drMdG1IFl3agK0p/pqT9KAo= sha512-2e8qq0ETcfWRI4HJBzQiA3UoyFk6tbNyG+qSaIBZLyW9Xf3sWZHN/lxe9fTh1U45DpPf07yj94KsUHHWe4Yk1A==" crossorigin="anonymous"></script>
    <!-- IE10 viewport hack for Surface/desktop Windows 8 bug -->
    <script src="/theme/js/ie10-viewport-bug-workaround.js"></script>
</body>
</html>